modules/desktop/graphics/labwc.config.nix

```diff
@@ -110,13 +109,13 @@ let
         <action name="Execute" command="${pkgs.brightnessctl}/bin/brightnessctl set 5%-" />
       </keybind>
       <keybind key="XF86_AudioRaiseVolume">
-        <action name="Execute" command="${audio-ctrl}/bin/audio-ctrl inc" />
+        <action name="Execute" command="bash -c 'PULSE_SERVER=audio-vm:4713 ${pkgs.pamixer}/bin/pamixer --unmute --increase 5'" />
```

Comment:
If using eww-volume here also is not possible i'd rethink this a bit.
Running bash -c is basically the same as running a script which needs some shell..
Maybe at least make a function for these almost duplicates?